packages/components/color-picker/src/color-picker.tsx

```diff
@@ -73,7 +73,23 @@ type ColorPickerOptions = {
 export type ColorPickerProps = ThemeProps<"ColorPicker"> &
   UseColorPickerProps &
   ColorPickerOptions
-
+ & Pick<
+   ColorSelectorProps,
+   | "withPicker"
+   | "withChannel"
+   | "saturationSliderRef"
+   | "saturationSliderProps"
+   | "swatchesProps"
+   | "withResult"
+   | "hueSliderRef"
+   | "hueSliderProps"
+   | "alphaSliderRef"
+   | "alphaSliderProps"
+   | "swatchesLabel"
+   | "swatches"
+   | "swatchesColumns"
+   | "channelProps"
+  >
```

Comment:
The type `channelProps` isn't included in type `ColorSelectorProps` ( from `color-selector.tsx`), so you may want to consider augmenting the typing as follows in `color-selector.tsx`:
```tsx
  export type ColorSelectorProps = ThemeProps<"ColorSelector"> &
    UseColorSelectorProps &
    ColorSelectorOptions &
    Pick<
      ColorSelectorBodyProps,
      | "withResult"
      | "withEyeDropper"
      | "eyeDropperRef"
      | "eyeDropperProps"
      | "hueSliderRef"
      | "hueSliderProps"
      | "alphaSliderRef"
      | "alphaSliderProps"
    > &
    Pick<
      ColorSelectorSwatchesProps,
      "swatchesLabel" | "swatches" | "swatchesColumns" | "swatchProps"
    > &
    ColorSelectorChannelsProps // add this
```
The type `ColorSelectorChannelsProps` includes the `channelProps` property.

And in color-selector.tsx, ensure that `channelProps` are received via its props, and then pass `channelProps` to <ColorSelectorChannels /> like this: 
```tsx
const {
    className,
    withResult = true,
    withPicker = true,
    withChannel = true,
    swatches,
    swatchesColumns = 7,
    inputProps,
    withEyeDropper,
    eyeDropperRef,
    eyeDropperProps,
    saturationSliderRef,
    saturationSliderProps,
    hueSliderRef,
    hueSliderProps,
    alphaSliderRef,
    alphaSliderProps,
    swatchesLabel,
    swatchProps,
    swatchesProps,
    channelProps, // add this
    ...computedProps
} = omitThemeProps(mergedProps)

// other codes...

return (
  // other codes...

  {withPicker && withChannel ? (
    <ColorSelectorChannels channelProps={channelProps} /> // add `channelProps`
  ) : null}

  // other codes...
)
```